openlabcmd/openlabcmd/plugins/jobs/image.py

```diff
@@ -4,10 +4,12 @@
 
 
 class ImagePlugin(Plugin):
-
     ptype = 'jobs'
     name = 'image'
 
+    def __init__(self, cloud, config):
+        super(ImagePlugin, self).__init__(cloud, config)
+
     def check(self):
         self.failed = False
```

Comment:
we already set the `failed` and `reasons` in parent class, we can remove these now.

Reply:
It's more like a `tag reset button`.